src/renderer/components/login/Landing.vue

```diff
@@ -2,22 +2,28 @@
   <section id="login--landing">
     <aph-icon name="logo"></aph-icon>
     <div class="version-number">v{{$store.state.version}}</div>
+    <!--<locale-changer></locale-changer>-->
```

Comment:
When we implement this, there is a design to follow. A login settings page that includes language and currency.

Reply:
I'll remove the comment, this was just a stub to test that it works. I will implement design when available.